### PR TITLE
[NDM] Disable logging in SNMP walk command

### PR DESCRIPTION
**Pull request by TCheruy**
### What does this PR do?
Logging was set info when running the SNMP walk command

### Describe how to test/QA your changes
Running `agent snmp walk`, verify no agent log are displayed. Output should be the same than the `snmpwalk` util.

### CODE REVIEW

File: cmd/agent/subcommands/snmp/command.go [hunk 1]
```diff
@@ -135,7 +135,7 @@ func Commands(globalParams *command.GlobalParams) []*cobra.Command {
 				fx.Supply(core.BundleParams{
 					ConfigParams: config.NewAgentParams(globalParams.ConfFilePath, config.WithExtraConfFiles(globalParams.ExtraConfFilePath), config.WithFleetPoliciesDirPath(globalParams.FleetPoliciesDirPath)),
 					SecretParams: secrets.NewEnabledParams(),
-					LogParams:    log.ForOneShot(command.LoggerName, "info", true)}),
+					LogParams:    log.ForOneShot(command.LoggerName, "off", true)}),
```
Comment (NouhaManai96) [inline cmd/agent/subcommands/snmp/command.go:138]: this was changed from `off` to `info` in #29148 , should probably check with @FlorianVeaux for the reasons behind this

**Review by NouhaManai96 — APPROVED** [review]

File: cmd/agent/subcommands/snmp/command.go (hunk 1 above)
Comment (FlorianVeaux) [reply cmd/agent/subcommands/snmp/command.go:138]: 👀  I think this is likely a testing oversight. I don't think we leverage the logger for printing anything to the console in this command

**Review by FlorianVeaux — APPROVED** [review]

File: cmd/agent/subcommands/snmp/command.go (hunk 1 above)
Comment (TCheruy) [reply cmd/agent/subcommands/snmp/command.go:138]: If we don't set it to off the output of `snmp walk` is poluted by logs like this unfortunately
```
2024-11-07 09:39:39 UTC | CORE | INFO | (pkg/aggregator/aggregator.go:762 in run) | Stopping aggregator
```

**Pull request merged**